_includes/foot.html

```diff
@@ -44,9 +45,9 @@ <h3>Social</h3>
                   <div class="logo-right-arm-f"></div>
               </div>
             </div>
-          </div>
-            <span class="label">gitter</span></a>
-          </li>
+          </div>-->
```

Comment:
Please delete L38 - L48 rather than commenting them out. They are of no use after this PR.

Reply:
okay